### DISCUSSION TITLE
Alpine base image version

### DISCUSSION THREAD

**Discussion by lukebakken** [docker-library/rabbitmq#596] [General]
With regard to https://github.com/docker-library/rabbitmq/pull/592

Why is `alpine:3.17` used and not `alpine:3`?

**Comment by tianon** [comment]
The build failures you're seeing on #589 are a great example -- it's the same as why we wouldn't pin to `debian:latest` or `ubuntu:latest`; we want to control when we do those stable updates to make sure that everything actually works correctly (and sometimes there are deeper concerns with distro upgrades, like https://github.com/docker-library/postgres/issues/582).

**Reply by lukebakken** [reply]
Thanks
